google/ggrc-core
# GGRC-819 :Fix UNIQUE constraints in the DB for slug

# Issue description

Fix UNIQUE constraints in the DB for slug

# Steps to test the changes

1. run the docker setup dev.
2.connect to DB.
3. Run below command and verify if all the columns have unique constraints or not.
SELECT table_name, column_name, COLUMN_DEFAULT, COLUMN_KEY FROM INFORMATION_SCHEMA.COLUMNS WHERE COLUMN_NAME = 'slug' AND TABLE_SCHEMA='ggrcdev';

# Solution description

Added migration version file, as per the requirement.
# Sanity checklist




# PR Review checklist
